lib/src/rules/avoid_dynamic_calls.dart

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import '../analyzer.dart';
+
+const _desc = r'Avoid method calls and property access on a "dynamic" target.';
```

Comment:
=> property accesses?

or =>

on "dynamic" targets.

?

Reply:
"method calls" and "property access" should either both be plural or neither be plural.